modules/developers-guide/pages/cli-reference/dfx-canister.adoc

```diff
@@ -669,6 +674,124 @@ Set "r7inp-6aaaa-aaaaa-aaabq-cai" as controller of "open_sf_assets".
 +
 You can now use the wallet canister `r7inp-6aaaa-aaaaa-aaabq-cai` to send cycles or add custodians to the `open_sf_assets` canister.
 
+== dfx canister send ==
+
+When creating your neuron stake, use the `+dfx canister send+` command with the `+dfx canister sign+` command when you want to separate these steps, rather than using the single `+dfx canister call+` command. Using separate calls can add security to the transaction.  
```

Comment:
```suggestion
Use the `+dfx canister send+` command after signing a message with the `+dfx canister sign+` command when you want to separate these steps, rather than using the single `+dfx canister call+` command. Using separate calls can add security to the transaction. 

For example, when creating your neuron stake, you might want to use the `+dfx canister sign+` command to create a signed `+message.json+` file using an air-gapped computer, then use the `+dfx canister send+` command to deliver the signed message.
```